trackers_integration/tests/test_trac.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2022-2024 Alexander Todorov <atodorov@otb.bg>
+# Copyright (c) 2025 Frank Sommer <Frank.Sommer@sherpa-software.de>
+#
+# Licensed under GNU Affero General Public License v3 or later (AGPLv3+)
+# https://www.gnu.org/licenses/agpl-3.0.html
+
+# pylint: disable=attribute-defined-outside-init, protected-access
+
+from django.utils import timezone
+
+from tcms.core.contrib.linkreference.models import LinkReference
+from tcms.rpc.tests.utils import APITestCase
+from tcms.testcases.models import BugSystem
+from tcms.tests.factories import ComponentFactory, TestExecutionFactory
+
+from trackers_integration.issuetracker import trac
+from trackers_integration.issuetracker.trac import Trac
+
+
+class TestTracIntegration(APITestCase):
+
+    @classmethod
+    def _fixture_setup(cls):
+        super()._fixture_setup()
+
+        cls.project_name = "Demo project"
+        cls.execution_1 = TestExecutionFactory()
+        cls.execution_1.case.summary = "Tested at " + timezone.now().isoformat()
+        cls.execution_1.case.text = "Given-When-Then"
+        cls.execution_1.case.save()  # will generate history object
+        cls.execution_1.run.summary = (
+            "Automated TR for Trac integration on " + timezone.now().isoformat()
+        )
+        cls.execution_1.run.save()
+
+        # 'kiwitcms-bot' user is authorized only for this project
+        cls.execution_1.build.version.product.name = cls.project_name
+        cls.execution_1.build.version.product.save()
+
+        cls.component = ComponentFactory(
+            name="Trac integration", product=cls.execution_1.build.version.product
+        )
+        cls.execution_1.case.add_component(cls.component)
+
+        cls.bug_system = (
+            BugSystem.objects.create(  # nosec:B106:hardcoded_password_funcarg
+                name="Trac for kiwitcms/test-trac-integration",
+                tracker_type="trackers_integration.issuetracker.Trac",
+                base_url="http://bugtracker.kiwitcms.org",
+                api_username="tester",
+                api_password="tester",
+            )
+        )
+        cls.integration = Trac(cls.bug_system, None)
+
+        # WARNING: container's certificate is self-signed
+        trac._VERIFY_SSL = False
+
+        create_params = {
+            "type": "defect",
+            "priority": "major",
+            "summary": "Smoke test failed",
+            "description": "Something went wrong",
+            "project": cls.project_name,
+            "component": cls.project_name,
+        }
+        issue = cls.invoke_trac_rpc("ticket.create", create_params)
```

Comment:
Should be the same as 

```suggestion
        issue = cls.integration.rpc.invoke_method("ticket.create", create_params)
```

and remove the invoke_trac_rpc method in the test class.